libs/pika/async_mpi/tests/unit/mpi_async_storage.cpp

```diff
@@ -62,6 +61,18 @@ namespace mpi = pika::mpi::experimental;
 namespace tt = pika::this_thread::experimental;
 namespace deb = pika::debug::detail;
 
+namespace pika::execution {
+    template <typename Sender>
+    auto async(Sender&& sender)
+    {
+        namespace ex = pika::execution::experimental;
+        auto sched = ex::thread_pool_scheduler{&pika::resource::get_thread_pool("default")};
+        auto snd = ex::schedule(sched) |
+            ex::then([sender = std::move(sender)]() mutable { ex::start_detached(std::move(sender)); });
+        ex::start_detached(std::move(snd));
+    }
+}    // namespace pika::execution
```

Comment:
I think it'd be clearer if you just replace the `just` with a `transfer_just` where you're using this right now. At least I don't like using the `pika::execution::async` name for this since it doesn't behave the same as `pika::async`.

Reply:
Please at least rename this to almost anything but `async`. Maybe `start_detached_on`?